automl/automl-script/script.whizzml

```diff
@@ -167,7 +206,7 @@
           [train* test*] (if (= "dataset" (resource-type valid))
                            [train valid]
                            (wait* (create-random-dataset-split train 0.8))))
-      (compute-important-fields train* test*))))
+      (compute-important-fields train* test* shallow-search))))
```

Comment:
again shallow-search or shallow....

Reply:
The script input is called `shallow-search`, but to avoid global variables, when I use it in a function I set it as a function argument and I name it just "shallow"

Reply:
In the cases you pointed, as they are not inside functions, I am using the global variable name, `shallow-search`